vpdq/cpp/hashing/filehasher.cpp

```diff
@@ -102,6 +104,15 @@ bool hashVideoFile(
           (int)fread_rc);
     }
   }
+  double secPerFrame = (double)duration / fno;
+  for (size_t i = 0; i < pdqHashes.size(); i++) {
+    vpdqFeature cur = pdqHashes[i];
+    pdqHashes[i] = (vpdqFeature){
```

Comment:
hmm, weird that you have to completely create a new object here.

Reply:
example does: `pdqHashes[i].timeStamp = pdqHashes[i].frameNumber * secPerFrame;` not work for you?

Reply:
It works. The previous issue is caused by using 
```
for (auto t:pdqHashes):
     t.timeStamp = t.frameNumber * secPerFrame;
```
It did not change the value.

Reply:
even simpler:
```  
for (auto& frame_feature : pdqHashes) {
    frame_feature.timeStamp = frame_feature.frameNumber * secPerFrame;
  }
```